src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md

```diff
@@ -40,30 +53,30 @@ To retrieve all properties that are stored in the `framework` namespace, use the
 
 ### Retrieve a single property from a namespace
 
-To retrieve a specific property from the `framework` namespace, specify the property name in the command by using the `-–name` flag. For example:
+To retrieve a specific property from the `framework` namespace, specify the property name in the command by using the `-–name` flag. For example, to retrive the credentials store property use the following command:
 
-```galasactl properties get --namespace framework --name propertyName```
+```galasactl properties get --namespace framework --name credentials.store```
  
 
 *Note:* The `-–name` flag cannot be used in conjunction with the `–suffix`, `--prefix`, or `-–infix` flags.
```

Comment:
`-suffix` should be `--suffix` (double-dash)

Reply:
why doesn't prefix come before suffix in your list ? Odd no ?

Reply:
added the -- and switched those values round